app/lib/common/widget/customAvatar.dart

```diff
@@ -0,0 +1,95 @@
+import 'dart:typed_data';
+import 'package:colorize_text_avatar/colorize_text_avatar.dart';
+import 'package:flutter/material.dart';
+import 'package:effektio/common/store/Colors.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+
+class CustomAvatar extends StatelessWidget {
+  final Future<List<int>> avatar;
```

Comment:
as said above, let's fix this external interface to require the specific FFI-Type and do all native conversion within the component please.